### ISSUE TITLE
Extendable Slash Commands with Custom User Commands

### ISSUE THREAD

**Issue by BigDog1400** [arikchakma/maily.to#36]
Right now, there isn't any way to add on to the commands that pop up when you type '/'. It might be great to have a feature that lets you do that

We could add a prop, let's call it 'slashCommands' (or any other name that sounds better). This prop can merge the default commands with any additional ones that the user wants.

I think we can make this happen by moving 'Command.configure({ suggestion })' over to extensions/index.ts and tweaking it a bit to 'Command.configure({ suggestion: getSlashCommandSuggestion })'.

**Comment by BigDog1400** [comment]
Let me know if you want me to tackle this, i can allocate some time this weekend to solve it

**Comment by arikchakma** [comment]
@BigDog1400 Yes, please carry on. I'll review and merge the `i18n` support asap. Maybe next weekend.

**Comment by arikchakma** [comment]
@BigDog1400 I'm gonna pick it up, what do you think?

**Comment by BigDog1400** [comment]
Hey, sorry, i didn't have the time to do it. Thanks for you work
